app/controllers/v1/user_controller.rb

```diff
@@ -3,27 +3,34 @@ class V1::UserController < ApplicationController
   before_action :downcase_email
 
   def show
-    render partial: 'v1/shared/user', locals: { user: current_user }
+    render partial: 'v1/shared/user', locals: {user: current_user}
   end
 
   def create
     @user = User.create(user_params.merge(
-      role: params[:role] || :member,
-      district_ids: extract_ids_from_params(:district_ids),
-      organization_ids: extract_ids_from_params(:organization_ids)
+        role: params[:role] || :member,
+        district_ids: extract_ids_from_params(:district_ids),
+        organization_ids: extract_ids_from_params(:organization_ids)
     ))
 
-    user_invitation = [UserInvitation, InventoryInvitation].map{|i| i.find_by(email: user_params[:email])}.compact.first
-    if user_invitation.present?
-      @user.errors.add(:base, "It seems you have already been invited. Please continue your registration here.")
+    user_invitation = [UserInvitation, InventoryInvitation].map { |i|
+      i.find_by(email: user_params[:email])
+    }.compact.first
+    invite_user = user_invitation.try(:user)
+
+    if invite_user && invite_user.sign_in_count > 0
```

Comment:
I've added some notes and did a bit of looking around.  We have the existing state in our code base in that there are still some invitations lurking about, and there are accounts with a lot of sessions logged (some over 100).